src/textual/screen.py

```diff
@@ -20,8 +20,8 @@ class Screen(Widget):
     CSS = """
     
     Screen {
-        layout: dock;
-        docks: _default=top;
+        layout: vertical;
+        overflow-y: auto;
```

Comment:
as discussed with Will: having a vertical layout by default in our Screens is a bit more intuitive than the Dock layout